app/models/pacbio/well.rb

```diff
@@ -73,16 +73,16 @@ def summary
       "#{sample_names} #{comment}".strip
     end
 
-    # return the sequencing_kit_box_barcode of this plate if well belongs to plate 1
+    # return the sequencing_kit_box_barcode of plate 1
     # used for 2-plate sample sheets
     def sequencing_kit_box_barcode_plate_1
-      plate.plate_number == 1 ? plate.sequencing_kit_box_barcode : nil
+      plate.run.plates.filter { |plate| plate.plate_number == 1 }.first&.sequencing_kit_box_barcode
     end
 
-    # return the sequencing_kit_box_barcode of this plate if well belongs to plate 2
+    # return the sequencing_kit_box_barcode of plate 2
     # used for 2-plate sample sheets
     def sequencing_kit_box_barcode_plate_2
-      plate.plate_number == 2 ? plate.sequencing_kit_box_barcode : nil
+      plate.run.plates.filter { |plate| plate.plate_number == 2 }.first&.sequencing_kit_box_barcode
```

Comment:
Maybe turn this into a method in well? e.g.` find_sequencing_kit_box_barcode(plate_number)`

Having to do this does feel like a code smell but not sure what the answer is at the moment.